astroquery/vo_conesearch/conesearch.py

```diff
@@ -223,32 +220,31 @@ def conesearch(center, radius, verb=1, catalog_db=None, pedantic=None,
         else:
             url = catalog['url']
 
-        cs.URL = url
-
         if verbose:  # pragma: no cover
             color_print('Trying {0}'.format(url), 'green')
 
         try:
-            r = cs.query_region(center, radius, verb=verb, cache=cache,
-                                verbose=verbose)
-
+            r = ConeSearch.query_region(
```

Comment:
can be done in follow-up, but would be nice to get rid of the single letter variables and rather just call this `result` or `response` or similar.

Reply:
:sweat_smile:  It is a valid point. I'll do it in a follow-up commit when a full review is given.

Reply:
Thinking about this more, I now prefer to revisit renaming short variables as a separate PR, just to make my life saner. 😬

Reply:
OK, opened: https://github.com/astropy/astroquery/issues/1572 for it.